docs/model/data-model.dbml

```diff
@@ -204,4 +217,14 @@ table files {
   submitted_at timestamp [not null]
   content bytea [not null]
   filename varchar(255)
-}
\ No newline at end of file
+}
+
+Table comments {
+  id bigint [pk, not null]
+  application_id bigint [not null, ref: > applications.id]
+  comments text
+  username text
```

Comment:
We need to capture the proper `user_id` which can be traced back to the user registration, but we also need to record the display name... so for consistency lets rename this to `user_name` and this should be `not null`.